reactor-pool/src/test/java/reactor/pool/InstrumentedPoolPoolMetricsTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2023 VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.pool;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InstrumentedPoolPoolMetricsTest {
+
+	@Test
+	void testIsInactiveForMoreThan() {
+		InstrumentedPool.PoolMetrics metrics = new TestPool(2);
+
+		assertThat(metrics.isInactiveForMoreThan(Duration.ofMillis(1999))).isTrue();
+		assertThat(metrics.isInactiveForMoreThan(Duration.ofSeconds(2))).isTrue();
```

Comment:
technically, 2s is not more than 2s, so I'd say this assertion should be false

Reply:
wasn't this implemented according javadoc?

```
		/**
		 * A convenience way to check the pool is inactive, in the sense that {@link #acquiredSize()},
		 * {@link #idleSize()}, {@link #pendingAcquireSize()} and {@link #allocatedSize()} are all at zero
		 * and that the last recorded interaction with the pool ({@link #secondsSinceLastInteraction()})
		 * was more than or exactly {@code duration} ago.
		 *
		 * @return true if the pool can be considered inactive (see above), false otherwise
		 * @see #secondsSinceLastInteraction()
		 */
```

```
that the last recorded interaction with the pool ({@link #secondsSinceLastInteraction()})
		 * was more than or exactly {@code duration} ago.
```

Reply:
You're right. Then it's not the implementation that is at fault, but the javadoc contradicts what the words in the method name say :(
`isInactiveForMoreThan` contradicts `more than or exactly `. In this case I have no more comments. If you're ok with this @violetagg please disregard my comment.